docs/IsolationNetworkPolicies.md

```diff
@@ -0,0 +1,210 @@
+# Isolation 
+
+_**Note:**_ below we start with Network Polices as an isolation method, later we will add other methods.
+
+_TODO_: add links to other options.
+
+## Introduction
+
+In the current Fybrik implementation the entry point of a data plane is published as a part of the `FybrikApplication` 
+status, it is completely unprotected, and as result everyone can connect to the data plane. If a Fybrik data plane 
+consist of a module chain, the same vulnerability exists for the chained modules.  
+In `Read` flow cases it leads to governance/security violations, `Write` and `Delete` flows can create severe accidents, 
+such as unauthorized data changes or deletions. Given that we have to provide a mechanism, which will prevent 
+unauthorized data plane access.
+
+## Challenge
+
+When we talk about Fybrik data plane isolation, we are talking about connectivity between a user workload, which is out of
+Fybrik scope, and Fybrik modules, which can be developed by third parties. Therefore, in this document (and in its
+extensions) we provide our recommendations how to protect data plane access and possible solutions. Responsibility of the
+data plane isolation is shared between Fybrik core, Fybrik users, module developers and probably IT teams.
+
+- Fybrik users should provide all relevant information about possible workload locations to Fybrik, correctly 
```

Comment:
Fybrik users should provide Fybrik with the information about possible workload locations and configure their workloads. For example, they need to ...

Reply:
done